linkstack/package.json

```diff
@@ -30,6 +35,7 @@
         "@walletconnect/keyvaluestorage": "^1.1.1",
         "axios": "^1.7.3",
         "lodash": "^4.17.21",
+        "plotly.js": "^2.35.1",
```

Comment:
The plotly.js should not be included. Is it possible to run on your localhost without it?

Reply:
When I try to install with yarn, that compilation problem appears, if I install ploty that problem disappears.